defaults/main.yml

```diff
@@ -62,3 +67,8 @@ alpaca_blueprint_settings:
     derivative_service_url: http://localhost:8000/homarus/convert
     error_max_redeliveries: 5
     camel_context_id: IslandoraConnectorHomarus
+  - pid: ca.islandora.alpaca.connector.fits
+    in_stream: activemq:queue:islandora-connector-fits
+    derivative_service_url: http://localhost:8050/fits/process
```

Comment:
Should this be port `8000` not `8050`?

Reply:
The microservice is configured to run on port 8050.  The Symfony 4 routing will clash with drupal's on 8000. The FITs ansible role (which we'll be releasing shortly) opens that port up for the microservice.

Reply:
Is this due to the FITS webservice as all the other microservices are running on port 8000 without clashing. Do we need to add a new virtualhost to the Apache config perhaps?